src/main/java/core/basesyntax/exception/UserValidationException.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.exception;
+
+public class UserValidationException extends Exception {
```

Comment:
```suggestion
public class RegistrationException extends Exception {
```

use more abstract name